lldb/source/Plugins/ObjectFile/Minidump/MinidumpFileBuilder.cpp

```diff
@@ -655,9 +655,10 @@ MinidumpFileBuilder::AddMemoryList(const lldb::ProcessSP &process_sp,
     const addr_t addr = core_range.range.start();
     const addr_t size = core_range.range.size();
     auto data_up = std::make_unique<DataBufferHeap>(size, 0);
+    Status read_error;
```

Comment:
The goal is to skip memory ranges that couldn't be read, without surfacing an error about them, right.  I don't mind it that much, but another way to be to use the existing `error` Status object (which we know is state==Success at this point), and if our memory read does fail, we could do 
```
if (error.Fail() || bytes_read == 0) {
      error.Clear();
      continue;
}
```

To make it clear that we don't want to surface a memory read failure for a region to the caller.  But this is more of a style preference I think.

Reply:
Yes, this also works. Let me know if you'd like me to change it

Reply:
I don't work on this plugin myself, I'm sure the way you expressed the idea is fine, it's little more than a style difference.